ElectronClient/app/gui/NotebookPropertiesDialog.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+const { _ } = require('lib/locale.js');
+const { themeStyle, buildStyle } = require('../theme.js');
+const DialogButtonRow = require('./DialogButtonRow.min');
+
+const Folder = require('lib/models/Folder.js');
+
+interface FolderPropertiesDialogProps {
+	theme: number,
+	folderId: string,
+	onClose: Function,
+}
+
+function styles_(props:FolderPropertiesDialogProps) {
+	return buildStyle('FolderPropertiesDialog', props.theme, (theme: any) => {
+		return {
+			folderLabel: {
+				...theme.textStyle,
+				flex: 1,
+				display: 'flex',
+				fontWeight: 'bold',
+				color: theme.color,
+			},
+			folderSaveButton: {
+				background: 'none',
+				border: 'none',
+			},
```

Comment:
Why remove the background and border? Buttons in general should simply have the theme.button style.

Reply:
I used [this line](https://github.com/laurent22/joplin/blob/639712c003b09d5652b5e25a681e93f6a29474f9/ElectronClient/app/gui/ShareNoteDialog.tsx#L48) as a reference. The only other Dialog reference I had was [this](https://github.com/laurent22/joplin/blob/639712c003b09d5652b5e25a681e93f6a29474f9/ElectronClient/app/gui/NotePropertiesDialog.jsx#L110), and it is definitely not useful whatsoever for trying to know what kind of style you want going forward. See [here](https://github.com/laurent22/joplin/pull/2450#issuecomment-583700940) please

Reply:
If you could attach screenshots of your dialog it would be easier to discuss all this. I still don't really understand what this style is for.

Reply:
![2020-02-08-055010_1690x1041_scrot](https://user-images.githubusercontent.com/59133880/74084687-fb1ea380-4a36-11ea-94ac-03a301eaa369.png)

There you go, although, are you saying you don't understand what my style code was for or what your own project's styling is all about? If the prior, it's completely because I was copying what was being used elsewhere. If the latter, that is very disheartening.